apps/cookbook/src/app/page/header/header.component.html

```diff
@@ -1,27 +1,124 @@
 <header>
-  <button
-    class="menu-button menu-button--align-right menu-button--only-mobile"
-    (click)="onToggleMenu()"
-  >
-    <svg class="icon"><use xlink:href="../../assets/icons/icon-sprite.svg#icon-burger"></use></svg>
-  </button>
+  <a [routerLink]="'/home/intro'">
+    <div class="kirby-logo">
+      <svg
```

Comment:
I am animating parts of the logo (added classes to some of the `g` tags - but it might still work if I extract it. I'll give it a go

Reply:
Looks like the cleanest approach is to inline the SVG as I have done - otherwise I have to include the animation styling in the SVG as a `<style>` tag (there are other options, but none that I deem better than inlining for this case)